server/src/main/java/org/opensearch/Version.java

```diff
@@ -71,9 +71,7 @@
     public static final Version V_EMPTY = new Version(V_EMPTY_ID, org.apache.lucene.util.Version.LATEST);
 
     public static final Version V_1_0_0 = new Version(1000099, org.apache.lucene.util.Version.LUCENE_8_8_2);
-    public static final Version V_1_0_1 = new Version(1000199, org.apache.lucene.util.Version.LUCENE_8_8_2);
```

Comment:
I missed that :(

Reply:
@dblock  should we bring it back? I could PR it shortly

Reply:
Let's let @adnapibar do it? :)

Reply:
sorry about that, I'll add that back
I removed based on `./gradlew verifyVersions` which will start failing once we add the version back.
And this is because we didn't release `1.0.1` to maven (https://repo1.maven.org/maven2/org/opensearch/opensearch/maven-metadata.xml) @dblock do you know if there was a reason why we didn't publish this version to maven?

Reply:
There were no changes in core for 1.0.1, see https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-1.0.1.md, so there are no new maven artifacts. Maybe we did the right thing in removing it from here then?

Reply:
Oh right, I remember that, so we did distribution 1.0.1 (plugin changes?) but kept core to 1.0.0, it was very confusing indeed, @adnapibar @dblock  I think we are fine here, the Opensearch server should never return 1.0.1 as its version/

Reply:
Yes I would think so, if there is no OpenSearch (core) 1.0.1 which is not in Maven then it doesn't make sense to include it as a released version. The reason we keep track of released versions is for backwards compatibility and 1.0.1 is same as 1.0.0.